sdk/appcenter-push/src/main/java/com/microsoft/appcenter/push/Push.java

```diff
@@ -202,15 +229,17 @@ private void enqueuePushInstallationLog(@NonNull String pushToken) {
      *
      * @param pushToken the push token value.
      */
-    synchronized void onTokenRefresh(@NonNull final String pushToken) {
-        AppCenterLog.debug(LOG_TAG, "Push token refreshed: " + pushToken);
-        post(new Runnable() {
-
-            @Override
-            public void run() {
-                enqueuePushInstallationLog(pushToken);
-            }
-        });
+    synchronized void onTokenRefresh(final String pushToken) {
```

Comment:
What's the reason for the change here, moving the non-null check inside the function?

Reply:
So that outside calls don't have to check